docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -0,0 +1,66 @@
+---
+id: deploy-process-application
+title: Deploy and run your process application
+description: You can deploy your process application to a Zeebe cluster, or run your process application to test and debug it, and observe how it performs in a live environment..
+---
+
+import DeployImg from './img/process-applications/define-stages-deploy.png'
+import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
+import DeployErrorImg from './img/process-applications/deploy-error.png'
+
+You can deploy your process application to a Zeebe cluster, and run your process application to test and debug it, and observe how it performs in a live environment.
+
+## Deploy your process application
+
+To deploy your process application:
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select **Deploy** to open the **Deploy process application** modal.
+   <p><img src={DeployImg} alt="Deploy a process application" /></p>
+1. Turn on the toggle for the cluster stage you want to deploy to.
```

Comment:
A minor comment: this feels verbose and hyper-specific. I don't have a concrete requirement or request for change so feel free to ignore it for now